dev/lint.py

```diff
@@ -1,55 +0,0 @@
-# coding: utf-8
```

Comment:
This file should be restored, unless you can articulate how you've replaced all of the functionality.